cmd/controller/controller.go

```diff
@@ -42,10 +44,12 @@ func init() {
 	if pp := os.Getenv(common.IMPORTER_PULL_POLICY); len(pp) != 0 {
 		pullPolicy = pp
 	}
-	glog.Infof("init: complete: CDI controller will create the %q version of the importer\n", importerImage)
+	glog.V(5).Infof("init: complete: CDI controller will create importer using image %q with verbosity %d\n", importerImage, verbose)
```

Comment:
define constants for the levels. Define only 3 levels named: VUser=1, VAdmin=2, VDebug=3